<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewCharWizardPage3</class>
 <widget class="QWizardPage" name="NewCharWizardPage3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="q5_groupBox">
     <property name="title">
      <string>5. Who is your lord, and what is your duty to them?</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTextEdit" name="nc3_q5_textEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="4">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Summary</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QLabel" name="summary_label">
        <property name="text">
         <string>SUMMARY</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="q6_groupBox">
     <property name="title">
      <string>6. What do you long for, and how might this impede your duty?</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTextEdit" name="nc3_q6_textEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="q7_groupBox">
     <property name="title">
      <string>7. What is your relationship with your clan?</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QRadioButton" name="nc3_q7_pos_radioButton">
          <property name="text">
           <string>Positive - +5 Glory</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QRadioButton" name="nc3_q7_neg_radioButton">
          <property name="text">
           <string>Conflicted - Choose a Skill</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" rowspan="3">
         <widget class="QTextEdit" name="nc3_q7_textEdit"/>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="nc3_q7_comboBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="q8_groupBox">
     <property name="title">
      <string>8. What do you think of Bushido?</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="1">
         <widget class="QRadioButton" name="nc3_q8_neg_radioButton">
          <property name="text">
           <string>Conflicted - Choose a Skill</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="nc3_q8_pos_radioButton">
          <property name="text">
           <string>Positive - +10 Honor</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="0" column="0" rowspan="5">
         <widget class="QTextEdit" name="nc3_q8_textEdit"/>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="nc3_q8_comboBox"/>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="nc3_q8_item_comboBox"/>
        </item>
        <item row="1" column="1">
         <widget class="QRadioButton" name="nc3_q8_mid_radioButton">
          <property name="text">
           <string>Divergent - Choose an item:</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
